hydroflow/src/lib.rs

```diff
@@ -22,6 +23,9 @@ pub use tokio_util;
 pub use tuple_list::tuple_list as tl;
 pub use tuple_list::tuple_list_type as tt;
 
+mod misc;
+pub use misc::*;
```

Comment:
isn't that `misc::*` bad namespace hygiene?

Reply:
Not the worst since it's re-exporting rather than actually using any of the imported items. Ill rename it to be more clear it's just macro exports